src/Cake/Arguments/ArgumentParser.cs

```diff
@@ -166,16 +166,6 @@ private bool ParseOption(string name, string value, CakeOptions options)
                 options.PerformDebug = ParseBooleanValue(value);
             }
 
-            if (name.Equals("mono", StringComparison.OrdinalIgnoreCase))
```

Comment:
@patriksvensson @mholo65 I am not sure if these should be removed completely.  I think we should deprecate these commands. i.e. leave them in just now, but have them only emit an error that the Mono scripting engine has been removed with an explanation as to why.  Then in a future version, we can remove them completely.  Thoughts?